workers/unity/Packages/io.improbable.gdk.core/NetworkStats/NetworkStatisticsSystem.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections.Generic;
+using Unity.Entities;
+
+namespace Improbable.Gdk.Core.NetworkStats
+{
+    [DisableAutoCreation]
+    public class NetworkStatisticsSystem : ComponentSystem
```

Comment:
This should run in the Initialize group, as the very first system
The Update should take all queued netstats from the previous frame, and store them for that frame (so that sampling doesn't miss any data)